com.chartboost.mediation/Runtime/Utilities/ChartboostMediationUtils.cs

```diff
@@ -0,0 +1,22 @@
+using System.Linq;
+using UnityEngine;
+
+namespace Chartboost.Utilities
+{
+    public static class ChartboostMediationUtils
+    {
+        public static Canvas GetCanvasWithHighestSortingOrder()
+        {
+            Canvas canvas = null;
+            foreach (var can in GameObject.FindObjectsOfType<Canvas>().OrderByDescending(x => x.sortingOrder))
```

Comment:
Ooof, this seems like a very badly optimized approach for Finding a Canvas. However, I cannot really think of any other ways to find a Canvas on the scene 😢.